internal/wrappers/projects.go

```diff
@@ -51,6 +51,7 @@ type ProjectsWrapper interface {
 	Update(projectID string, model *Project) error
 	Get(params map[string]string) (*ProjectsCollectionResponseModel, *ErrorModel, error)
 	GetByID(projectID string) (*ProjectResponseModel, *ErrorModel, error)
+	GetByName(name string) (*ProjectResponseModel, *ErrorModel, error)
```

Comment:
you didn't use this func from the code

Reply:
It is used by "deleteProjectByName"

Reply:
we don't add code only for tests (only if it mast mast)